sources/packages/backend/apps/api/src/route-controllers/application/models/application.dto.ts

```diff
@@ -178,6 +178,9 @@ export class EnrolmentApplicationDetailsAPIOutDTO {
   firstDisbursement: DisbursementDetailsAPIOutDTO;
   secondDisbursement?: DisbursementDetailsAPIOutDTO;
   assessmentTriggerType?: AssessmentTriggerType;
+  verifiedDisabilityStatus?: boolean;
+  hasValidMSFAAStatus?: boolean;
+  hasRestriction?: boolean;
```

Comment:
As the expectation of the tracker is on completed applications, the new properties must be added onto `CompletedApplicationDetailsAPIOutDTO`